modules/ROOT/pages/6.1-release-notes.adoc

```diff
@@ -0,0 +1,1121 @@
+= TinyMCE 6.1
+:navtitle: TinyMCE 6.1
+:description: Release notes for TinyMCE 6.1
+:keywords: releasenotes, new, changes, bugfixes
+
+include::partial$misc/admon-releasenotes-for-stable.adoc[]
+
+[[overview]]
+== Overview
+
+{productname} 6.1 was released for {enterpriseversion} and {cloudname} on Wednesday, July 13^th^, 2022. It includes {productname} 6.1 and additional changes to premium plugins. These release notes provide an overview of the changes for {productname} 6.1:
+
+* xref:accompanying-premium-plugin-changes[Accompanying Premium Plugin changes]
+* xref:accompanying-premium-skins-and-icon-packs-changes[Accompanying Premium Skins and Icon Packs changes]
+* xref:bug-fixes[Bug fixes]
+* xref:changes[Changes]
+* xref:improvements[Improvements]
+* xref:additions[Additions]
```

Comment:
This seems like all the content here is out of order/backwards... the structure we've always used at Tiny is: `New features` -> `Improvements` -> `Changes` -> `Bug fixes`. There's marketing reason behind that as well, since it's new features that are enticing to potential customers.

- [ ] Fix the release notes order so that it follows the structure we're supposed to use